### ISSUE TITLE
o-share: improve label for links - ID: DAC_Share_Horizontal_01

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/origami#557]
# Issue
This is from page 185 of the [audit report](https://drive.google.com/file/d/17wq6b0CIX41RA4muBFDzUDxE0-qzHBIH/view)

Issue can be found on: <https://www.ft.com/__origami/service/build/v3/demo?component=o-share@8.0.0&demo=horizontal&system_code=origami-repo-data&brand=master>

The links in the share element are non-descriptive and do not inform the user of their purpose. Screen reader users may think the links are to visit social media sites as opposed to sharing to the sites.
The links also open in a new window but fail to inform users of this behaviour. This issue may particularly affect screen reader users who may become disorientated by the change in behaviour.

# Screen reader comments:

> “The social media links each open in a new window, but I was not informed of this behaviour. This behaviour may confuse some Screen Reader users as it is not immediately apparent that users will need to use the Alt + TAB combination to return to the previous window. This is because the social media page opens in a separate instance of the web browser, making the Alt + left arrow’ combination to go back to the previous page redundant.
> It would benefit users if this information was presented in the link text. Alternatively, if the link was opened in the same window, they could then use the Alt + left arrow’ combination to go back to the previous page.”
> “Testing the social media links using both VoiceOver and TalkBack, I found that they opened in a new tab. Unfortunately, this was not apparent from the link text.
> Informing users that the link opens in a new tab would help to avoid confusion and ensure that users do not become disorientated by the appearance of the new tab.”

# Additional instances:
Additional instances of this issue exist on other pages throughout the website; wherever this issue occurs, they too will need to be resolved.
URL: https://www.ft.com/__origami/service/build/v3/demo?component=o-share%408.0.0&demo=small&system_code=origami-repo-data&brand=master
Journey 52.
URL: https://www.ft.com/__origami/service/build/v3/demo?component=o-share%408.0.0&demo=inverse&system_code=origami-repo-data&brand=master
Journey 53.

# Proposed Solution:
Add more description to the links which describes that the links are to share to social media.

# Change on request:
Add visually-hidden text to the links to inform screen reader users that they open in a new window.
Example:
```html
<li class="o-share__action">
	<a class="o-share__icon o-share__icon--twitter" href="https://twitter.com/intent/tweet?url=http%3A//on.ft.com/1mUdgA2&amp;text=Pfizer%20says%20its%20AstraZeneca%20vow%20over%20big%20UK%20presence%20is%20binding&amp;related=ftcompanies&amp;via=FT" rel="noopener">
		<span class="o-share__text">Share on Twitter<span class="visually- hidden">(opens in a new tab)</span></span>
	</a>
</li>
```
